### PR TITLE
HAAR-470: Added new download endpoint for beta search

### PR DESCRIPTION
**Pull request by ravmoj**
Added new download endpoint for beta search

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/repository/UserPersonDetailRepositoryTest.kt [hunk 1]
```diff
@@ -791,6 +791,7 @@ class UserPersonDetailRepositoryTest {
           "IBRAGIM.MIHAIL",
           "MARIAN.CHESED",
           "LEOPOLDO.CHESED",
+          "LEOPOLDO.CHESED",
```
Comment (Mjwillis) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/repository/UserPersonDetailRepositoryTest.kt:794]: doesn't this mean we have a duplicate again?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResourceIntTest.kt [hunk 2]
```diff
@@ -814,12 +814,23 @@ class UserResourceIntTest : IntegrationTestBase() {
           .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_ACCESS_ROLES"), user = "jane.lsa.wwi"))
           .exchange()
           .expectStatus().isOk
-          .expectBody()
-          .jsonPath("$.numberOfElements").isEqualTo(1)
-          .jsonPath("$.content[0].dpsRoleCount").isEqualTo(0)
-          .jsonPath("$.content[0].activeCaseload.id").isEqualTo("BXI")
-          .jsonPath("$.content[0].activeCaseload.name").isEqualTo("Brixton (HMP)")
-          .jsonPath(matchByUserName, "marco.rossi").exists()
+          .expectBody().json(
```
Comment (Mjwillis) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResourceIntTest.kt:817]: why changed?

**Review by Mjwillis — APPROVED** [review]
Couple of comments

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/repository/UserPersonDetailRepositoryTest.kt (hunk 1 above)
Comment (ravmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/repository/UserPersonDetailRepositoryTest.kt:794]: yes, it uses the existing criteria hence reverted the test. Will create a ticket to address duplicates

**Pull request merged**